rosette_apiExamples/categories.cs

```diff
@@ -11,14 +11,22 @@ namespace rosette_apiExamples
     class categories
     {
         /// <summary>
-        /// Example code to call Rosette API to get a document's (located at given URL) category.
+        /// Example code to call Rosette API to get a document's (located at given URL) categories.
         /// Requires Nuget Package:
         /// rosette_api
         /// </summary>
-        static void Main()
+        static void Main(string[] args)
         {
             //To use the C# API, you must provide an API key
-            CAPI CategoriesCAPI = new CAPI("your API key");
+            string apikey = "Your API key";
+            
+            //You may set the API key via command line argument:
+            //categories yourapikeyhere
+            if (args.Length == 0)
+            {
+                apikey = args[0];
```

Comment:
is this c# special? In other languages args[0] would be undefined if array length is 0 ...